README.md

```diff
@@ -590,71 +671,6 @@ Here is a sample app.config file that uses [log4net](http://logging.apache.org/l
   </log4net>
 ```
 
-Getting the code coverage
```

Comment:
Why you removed this section? This is needed to get the code coverage of the driver.

Reply:
commit blip. Content is there